### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
in this branch we:

- Added CSS styling using Tailwind

### CODE REVIEW

File: src/components/Nav.js [hunk 1]
```diff
@@ -5,24 +5,33 @@ import {
...
+          </li>
+          <li className="nav-item px-3 py-2 font-mono flex items-center text-gray-400 text-lg hover:opacity-75">
+            <Link to="/Category">Category</Link>
+          </li>
+        </ul>
+        <div className="flex flex-col lg:flex-row list-none ">
+          <li className="p-3 border border-slate-300 text-right rounded-full text-blue-500 font-md"><HiUser /></li>
+        </div>
+      </nav>
+
```
Comment (Wes-Isaac) [inline src/components/Nav.js]: - [ ] Great job so far, your nav-bar design is slightly different than the one provided, kindly make sure spacing color and fonts are implemented correctly.

### Your Website
Screen Shot 2022-09-24 at 3 16 15 PM

### Zeplin Template
Screen Shot 2022-09-24 at 3 17 45 PM

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Kwaleyela-Ikafa, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- No linter errors 👍.
- Style implemented(95%) 🚀.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Nav.js (hunk 1 above)
Comment (Kwaleyela-Ikafa) [reply src/components/Nav.js]: @Wes-Isaac thanks for pointing that, i tried to the best of my ability to fix the issue

**Review by phelian23 — APPROVED** [review]
Hi @Kwaleyela-Ikafa,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good descriptive PR 👍
✅ Good commit messages 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**